src/frontend/screens/Onboarding/walletSelection/screens/import/components/BrowsersAndManualImportOptions/index.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { useState } from 'react'
+import { BrowsersAndManualImportOptionsProps } from '../../types'
+import { BrowserPackageManagerImportOption } from '../BrowserPackageManagerImportOptions'
+import { Button } from '@hyperplay/ui'
+import styles from './index.module.scss'
+import { t } from 'i18next'
+
+export function BrowsersAndManualImportOptions({
+  importOptions,
+  handleImportMmExtensionClicked,
+  browserSelected,
+  setBrowserSelected
+}: BrowsersAndManualImportOptionsProps) {
+  const [showManualImport, setShowManualImport] = useState(false)
+  const [manualImportPath, setManualImportPath] = useState(
+    '<profile>/Local Extension Settings/<metamask app id>'
+  )
+
+  function getPackageManagersForBrowser() {
+    return Object.keys(importOptions[browserSelected]!)
+  }
+
+  return (
+    <>
+      {
+        <div className={styles.profileOptionsContainer}>
+          {getPackageManagersForBrowser().map((pkgManager) => {
+            if (importOptions[browserSelected]![pkgManager].length === 0)
+              return null
```

Comment:
For readability, lets at least put another new line under the `return null` or we wrap it with curly brackets

Reply:
sure I'll add brackets but we should not enforce linting styles through code review. It wastes time. If it passes the linter, it should be okay 

Here is the issue for adding eslint curly to enforce this at the linting level https://github.com/HyperPlay-Gaming/hyperplay-desktop-client/issues/581